components/series/SeriesTable.vue

```diff
@@ -406,10 +413,14 @@ export default class SeriesTable extends Vue {
     if (vol === 0 || !parseFloat(String(vol)) || !isFinite(vol)) {
       return '---'
     }
-    const volumePercent = Math.round(vol * 100) / 100
-    return volumePercent > 0
-      ? `<div style="color: #41b883"> +${volumePercent}%</div>`
-      : `<div class="has-text-danger"> ${volumePercent}%</div>`
+    this.volumePercent = Math.round(vol * 100) / 100
+    return this.volumePercent > 0
+      ? `+${this.volumePercent}%`
+      : `${this.volumePercent}%`
+  }
+
+  public getClassForVolume() {
+    return this.volumePercent > 0 ? 'has-text-increase' : 'has-text-danger'
   }
```

Comment:
What will happen if volumePercent === 0 or is undefined? we will be seing '---' with red text?

Reply:
I'm having issue while loading series-insight page

Reply:
yeah me too, on local, so wasn't able to test it

Reply:
and yes it will return '---'

Reply:
we have some issues with a subquery (indexer) and once we deploy subsquid indexer, we should get back these big queries, hold on patience :D we've crossed their limits with size of our payload, well happens.